### PR TITLE
Unpon loading schema, always ensure bde schema permissions

### LINKED ISSUE #165: Ensure bde schema is fully setup by sql scripts and schema loader, even if `bde` schema already exists.
At the moment, the 02-bde_schema.sql file, which is responsible for creating tables and granting permissions to roles, simply does NOTHING when a `bde` schema already exists, giving up any attempt to ensure required tables/permission are present instead.

This was a problem for tests in #162

### PR DESCRIPTION
**Pull request by strk**
Rather than doing nothing when a bde schema is found.
Closes #165

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 All notable changes for the LINZ BDE schema are documented in this file.
 
 ## 1.8.0dev - 2019-MM-DD
-### Added
+### Enhanced
+- Schema loaders will now recover permissions (#165)
```
Comment (imincik) [inline CHANGELOG.md]: * @strk , would you please change this changelog message to something more clear ? Thanks

**Comment by strk** [comment]
> * @strk , would you please change this changelog message to something more clear ? Thanks

Do you like eb63a9b better ?

File: CHANGELOG.md (hunk 1 above)
Comment (imincik) [reply CHANGELOG.md]: Little bit better.

File: sql/02-bde_schema.sql [hunk 2]
```diff
@@ -49,56 +40,39 @@ CREATE TABLE bde.crs_action (
...
-    ADD CONSTRAINT pkey_crs_action PRIMARY KEY (audit_id);
-
 ALTER TABLE bde.crs_action ALTER COLUMN tin_id SET STATISTICS 500;
 ALTER TABLE bde.crs_action ALTER COLUMN att_type SET STATISTICS 500;
 ALTER TABLE bde.crs_action ALTER COLUMN ste_id SET STATISTICS 500;
 ALTER TABLE bde.crs_action ALTER COLUMN audit_id SET STATISTICS 500;
 
 ALTER TABLE bde.crs_action OWNER TO bde_dba;
 
-REVOKE ALL ON TABLE bde.crs_action FROM PUBLIC;
```
Comment (imincik) [inline sql/02-bde_schema.sql]: * where did you moved these `revoke` and `grant` statements ?

Comment (imincik) [reply sql/02-bde_schema.sql]: I see now. Looks like previously GH didn't rendered full diff for me. This comment is solved now.

**Pull request merged**